CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change.
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
```

Comment:
```suggestion
Please make sure to follow our code of conduct in all your interactions with the project.
```

Reply:
Ok, I'll make the necessary changes in a while ...

**Thanks for reviewing the documentation**